src/compiler/rust/canister/src/ic/print.rs

```diff
@@ -1,13 +1,19 @@
+use std::convert::TryInto;
+
 use quickjs_wasm_rs::{CallbackArg, JSContextRef, JSValueRef};
 
 pub fn native_function<'a>(
     context: &'a JSContextRef,
     _this: &CallbackArg,
     args: &[CallbackArg],
 ) -> Result<JSValueRef<'a>, anyhow::Error> {
-    for arg in args {
-        let value = arg.to_js_value()?;
-        ic_cdk::println!("{:?} ", value);
-    }
+    let string_to_print: String = args
+        .get(0)
+        .expect("console.log must have at least one argument")
```

Comment:
`console.log()` (without params) is valid in JS. Just make sure that still works.

Reply:
Addressed